airflow/utils/net.py

```diff
@@ -38,6 +38,14 @@ def get_hostname():
     if not callable_path:
         return socket.getfqdn()
 
+    # Use IP address if callable_path is configured as "use_ip_address"
+    # Fallback to socket.getfqdn() if there is error.
+    if callable_path == "use_ip_address":
```

Comment:
Yes to the feature, but I'm not 100% sure this is the right way of doing it -- I feel it would be easier/less confusing to put a function in airflow.utils.net like

```python
def host_ip_address()
    socket.gethostbyname(socket.getfqdn())
```

And then we can tell people to use `airflow.utils.net:host_ip_address` as the config value

Reply:
Good idea. Let me address accordingly. Thanks @ashb

Reply:
Hi @ashb , updated as suggested. Named the new function as `get_host_ip_address()` though. PTAL.